PoC/kafka-pull/README.md

```diff
@@ -1,41 +1,66 @@
-| `topic`                   | Defines the Kafka topic                                                                                             | `true`    |
-| `kafka.bootstrap.servers` | Defines a custom endpoint URL to kafka                                                                              | `true`    |
-| `kafka.sasl.mechanism`    | Defines the SASL kafka mechanism (SCRAM-SHA-256 or SCRAM-SHA-512)                                                   | `true`    |
-| `kafka.security.protocol` | Defines the kafka  security protocol (SASL_PLAINTEXT or SASL_SSL)                                                   | `true`    |
-| `pollDuration`            | Defines the duration of the consumer polling. The value should be a ISO-8601 duration e.g. "PT10S" for 10 seconds.  | `false`   |
-| `secretKey`               | Defines the `vault` entry containing the secret token/credentials                                                   | `true`    |
-| `groupPrefix`             | Defines the groupPrefix that will be allowed to use for the consumer                                                | `true`    |
+## DataAddress Schema
 
-#### Secret Resolution
+When creating a Kafka asset, use the following properties in the DataAddress:
 
-The `secretKey` property should point to a `vault` entry that contains a string object with admin kafka credentials.
+| Key                       | Description                                                                                                        | Mandatory |
+|:--------------------------|:-------------------------------------------------------------------------------------------------------------------|-----------|
+| `type`                    | Identifier of kafka data address. Should be set as 'KafkaBroker'                                                   | `true`    |
+| `topic`                   | Defines the Kafka topic                                                                                            | `true`    |
+| `kafka.bootstrap.servers` | Defines a custom endpoint URL to kafka                                                                             | `true`    |
+| `kafka.sasl.mechanism`    | Defines the SASL kafka mechanism (OAUTHBEARER)                                                                     | `true`    |
+| `kafka.security.protocol` | Defines the kafka security protocol (SASL_PLAINTEXT or SASL_SSL)                                                   | `true`    |
+| `pollDuration`            | Defines the duration of the consumer polling. The value should be a ISO-8601 duration e.g. "PT10S" for 10 seconds. | `false`   |
+| `groupPrefix`             | Defines the groupPrefix that will be allowed to use for the consumer                                               | `true`    |
+| `tokenUrl`                | The OAuth token URL for retrieving access tokens                                                                   | `true`    |
```

Comment:
one kind of variable formatting . or Camel Case please not both.

Reply:
The dot notation was used to have a direct visual distinction between properties the Kafka client needs and properties which are used by the EDC extension internally. I liked it this way and would like to keep it.